### DISCUSSION TITLE
Multiple Data Destinations for different projects/repositories in a Single Harbor Instance?

### DISCUSSION THREAD

**Discussion by keremsan** [goharbor/harbor#21070] [Q&A] [answered]
I'm interested in understanding if there is a way to configure multiple data destinations (such as different storage backends) for different projetas/repos within the same instance?

For example:
- Projects A and B -> /storage-main/...
- Project X -> /storage-x/...
- Project Y -> /storage-y/...

The reason for this setup is that Project X and Y are expected to grow significantly larger than the other projects, and I would like to allocate a different file destination for different projects (can be considered like sharding on filesystem) 

Thanks in advance

**Comment by Vad1mo** [comment, marked as answer]
No this is not possible with Harbor, I recommend using S3 as the storage layer over file system.
